targetd.te

```diff
@@ -48,6 +49,8 @@ fs_manage_configfs_files(targetd_t)
 fs_manage_configfs_lnk_files(targetd_t)
```

Comment:
Why is access to unlabeled_t required? Which directory it is, which filesystem, do you have complete audit records?

Reply:
See https://github.com/fedora-selinux/selinux-policy-contrib/pull/334#issuecomment-694997933